.github/workflows/linux-32bit-build-and-test.yaml

```diff
@@ -80,10 +80,11 @@ jobs:
       shell: bash
       run: |
         if compgen -G "/tmp/core*" > /dev/null; then
-          apt-get install postgresql-${{ matrix.pg_major }}-dbgsym >/dev/null
+          PG_MAJOR=$(echo "${{ matrix.pg }}" | sed -e 's![.].*!!')
+          apt-get install postgresql-${PG_MAJOR}-dbgsym >/dev/null
```

Comment:
It makes more sense to add it to the matrix above. It seems to have been there before, so not sure why it is not present any more.

Reply:
I followed the approach used elsewhere in the file. Not sure it can be added to the matrix since a major version is not a valid option for the matrix and needs to be computed. (We only have run full/canonical version numbers.)

I guess one option would be to have a previous step to compute the major version and then set an environment variable to use in other steps. But that is a bigger refactoring, which I think is outside the scope of this PR. 

I just wanted the CI to work for the rest of the PR.

Reply:
Actually, if were to do this, I think it fits better in a separate refactoring PR since it affects other lines in the yaml that aren't touched by this PR.

Reply:
Either way is fine. I approved the PR because my only concern were the tests and whether you should add that, but you have already answered it.